app/controllers/field_slips_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class FieldSlipsController < ApplicationController
+  include Show
```

Comment:
Tried moving `def show` into the show.rb file, but got Rubocop complaints.  Not sure what that's all about.